src/index.html

```diff
@@ -5,10 +5,57 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>MOYO header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img
+        src="images\logo.png"
+        alt="logo"
+        class="logo">
+      </a>
+      <nav class="nav">
```

Comment:
Don't forget
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/e5a05b3d-68bf-4c04-9f85-bd6afcd7ee19)

```suggestion
      </a>
      
      <nav class="nav">
```